Packs/Sigma/Playbooks/playbook-Sigma_-_Query_using_Sigma_rules.yml

```diff
@@ -0,0 +1,1344 @@
+          "y": 1505
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "29":
+    id: "29"
+    taskid: efabe847-b0ed-4422-8b38-c69cf49ba387
+    type: title
+    task:
+      id: efabe847-b0ed-4422-8b38-c69cf49ba387
+      version: -1
+      name: Done
+      type: title
+      iscommand: false
+      brand: ""
+      description: ''
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 1125,
+          "y": 1710
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "31":
+    id: "31"
+    taskid: 9278ba3e-09fd-4aac-8b82-13085eb193b9
+    type: title
+    task:
+      id: 9278ba3e-09fd-4aac-8b82-13085eb193b9
+      version: -1
+      name: Done
+      type: title
+      iscommand: false
+      brand: ""
+      description: ''
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 260,
+          "y": 1140
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "32":
+    id: "32"
+    taskid: 07124b2c-af3d-448c-8ca7-de8feba71c1d
+    type: title
+    task:
+      id: 07124b2c-af3d-448c-8ca7-de8feba71c1d
+      version: -1
+      name: Done
+      type: title
+      iscommand: false
+      brand: ""
+      description: ''
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 1125,
+          "y": 1140
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "33":
+    id: "33"
+    taskid: 097dfcc1-fd32-442e-8215-8827b015e948
+    type: title
+    task:
+      id: 097dfcc1-fd32-442e-8215-8827b015e948
+      version: -1
+      name: Done
+      type: title
+      iscommand: false
+      brand: ""
+      description: ''
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 1990,
+          "y": 1140
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+view: |-
+  {
+    "linkLabelsPosition": {},
+    "paper": {
+      "dimensions": {
+        "height": 1725,
+        "width": 2530,
+        "x": 50,
+        "y": 50
+      }
+    }
+  }
+inputs:
+- key: Query
+  value:
+    simple: type:"Sigma Rule"
+  required: false
+  description: ""
+  playbookInputQuery:
+inputSections:
+- inputs:
+  - Query
```

Comment:
Can it be replaced with the 'From Indicators'?
![image](https://github.com/user-attachments/assets/18694d95-6e10-46ef-9a3d-bce823c284a4)

Reply:
I think there was a reason to remove it from there as something didnt work well with it there but cant remember exactly why. I think it had to do with the fields being stored.